configs/dest/README.md

```diff
@@ -0,0 +1,49 @@
+# DEST
+
+[DEST: Depth Estimation with Simplified Transformer](https://arxiv.org/abs/2204.13791)
+
+## Introduction
+
+<!-- [ALGORITHM] -->
+
+<a href="https://github.com/NVIDIA/DL4AGX/tree/master/DEST">Official Repo</a>
```

Comment:
Add something like "DEST was developed for dense estimation tasks such as semantic segmentation and depth estimation. The core of the method is the simplified attention that has achieved better accuracy and latency as compared to the standard attention mechanism". Please find more information on official repo.

Reply:
Can you add this to introduction? The reason is that the paper's title is depth, and we want to divert from that narrow scope by explaining it's for dense prediction